cmd/kubeadm/app/constants/constants.go

```diff
@@ -180,8 +180,8 @@ const (
 	DiscoveryRetryInterval = 5 * time.Second
 	// PatchNodeTimeout specifies how long kubeadm should wait for applying the label and taint on the control-plane before timing out
 	PatchNodeTimeout = 2 * time.Minute
-	// TLSBootstrapTimeout specifies how long kubeadm should wait for the kubelet to perform the TLS Bootstrap
-	TLSBootstrapTimeout = 5 * time.Minute
+	// kubeletTLSBootstrap specifies how long kubeadm should wait for the kubelet to perform the TLS Bootstrap
+	KubeletTLSBootstrap = 5 * time.Minute
```

Comment:
is this rename needed?

Reply:
It is not need to be renamed. But we try to add "KubeletTLSBootstrap" to set  the kubelet bootstrap timeout be configurable, and default value of KubeletTLSBootstrap is come from TLSBootstrapTimeout . So I try to rename TLSBootstrapTimeout  to KubeletTLSBootstrap  for conformance.